src/docs/markdown/caddyfile/options.md

```diff
@@ -106,35 +106,57 @@ Possible options are:
 }
 ```
 
+
 ## General Options
 
 ##### `debug`
-Enables debug mode, which sets all log levels to debug (unless otherwise specified).
+Enables debug mode, which sets the log level to `DEBUG` for the default logger. This may reveal more information about Caddy's internals. Enable this before asking for help on the [community forums](https://caddy.community)!
+
 
 ##### `http_port`
 The port for the server to use for HTTP. For internal use only; does not change the HTTP port for clients. Default: `80`
 
+
 ##### `https_port`
 The port for the server to use for HTTPS. For internal use only; does not change the HTTPS port for clients. Default: `443`
 
+
 ##### `order`
-Sets or changes the standard order of HTTP handler directive(s). Can set directives to be `first` or `last`, or `before` or `after` another directive.
+Sets or changes the [standard order](/docs/caddyfile/directives#directive-order) of HTTP handler directive(s). Can set directives to be ordered `first` or `last`, or inserted `before` or `after` another directive.
+
+For example, to make sure the [`replace-response` plugin's](https://github.com/caddyserver/replace-response) directive is ordered after `encode`, to make sure it manipulates the response body before compression:
+
+```caddy-d
+order replace after encode
+```
+
 
 ##### `storage`
 Configures Caddy's storage mechanism. The default is [`file_system`](/docs/json/storage/file_system/). There are many other available [storage modules](/docs/json/storage/) provided as plugins.
 
+For example, to change the file system's storage location:
+
+```caddy-d
+storage file_system /path/to/custom/location
+```
+
+Changing the storage location is typically done when needing to sync Caddy's storage across multiple instances of Caddy, to make sure they all use the same certificates and keys. See the [Automatic HTTPS section on storage](/docs/automatic-https#storage) for more details.
+
+
 ##### `storage_clean_interval`
 How often to scan storage units for old or expired assets and remove them. These scans exert lots of reads (and list operations) on the storage module, so choose a longer interval for large deployments. The value is a [duration value](/docs/conventions#durations). Default: 24h.
 
 Storage will always be cleaned when the process first starts. Then, a new cleaning will be started this duration after the previous cleaning started if the previous cleaning finished in less than half the time of this interval (otherwise next start will be skipped).
 
+
 ##### `admin`
-Customizes the [admin API endpoint](/docs/api). If `off`, then the admin endpoint will be disabled. If disabled, config changes will be impossible without stopping and starting the server.
+Customizes the [admin API endpoint](/docs/api). If `off`, then the admin endpoint will be disabled. If disabled, config changes will be impossible without stopping and starting the server. Default: `localhost:2019`.
```

Comment:
Not sure I love documenting the default in two separate places, we already link to the canonical docs page for the API too.
```suggestion
Customizes the [admin API endpoint](/docs/api). If `off`, then the admin endpoint will be disabled. If disabled, config changes will be impossible without stopping and starting the server.
```

Reply:
Everything else documents their defaults though :man_shrugging: I think it's helpful to mention it here in case they land here before `/docs/api`, or don't even go to that page. But point taken. I can go either way on this one.